### DISCUSSION TITLE
interaction.get_open_filename_input crashed in Mac M1

### DISCUSSION THREAD

**Discussion by baikaishiuc** [Vector35/binaryninja-api#3577] [Q&A] [answered]
I try same version BN in different OS windows and mac m1 

**Binary Ninja Version:** 
Version 3.1.3469 Personal (Build ID 93650e26)

**Describe the bug**
1. right click IL Panel, popup menu, Plugin -> PBD -> Set Symbol Path -> chonse arbitary path, click ok, now crashed in mac m1 ,but windows is ok.
2. or you can write a plugin, register it
```python
from binaryninja import *
from pathlib import Path
from time import sleep

def paint_trace(bv:BinaryView,function):
	trace_filename= interaction.get_open_filename_input("Filename:")
	return

PluginCommand.register_for_address("PaintTrace", "Color Trace Instruction", paint_trace)
```
repeat step like 1, then BN exit

**Addition context:**
I found BN about page suggest update QT, do I need update it ?
<img width="531" alt="image" src="https://user-images.githubusercontent.com/618485/197166226-945636d0-e41e-4c75-877c-13e810456156.png">

**Comment by psifertex** [comment, marked as answer]
Sorry for not replying sooner but this was resolved some time ago on the latest version of BN. If you update to either a recent dev or the latest stable it should be fixed.

**Reply by baikaishiuc** [reply]
thank you,man
